core/trino-main/src/main/java/io/trino/execution/buffer/OutputBufferInfo.java

```diff
@@ -111,9 +115,15 @@ public long getTotalPagesSent()
         return totalPagesSent;
     }
 
+    @JsonProperty
+    public TDigestHistogram getUtilization()
+    {
+        return utilization;
+    }
+
     public OutputBufferInfo summarize()
```

Comment:
See: https://github.com/trinodb/trino/commit/d5c7c76ef5e5b62631c310f041ccb7469ed57c52
Histogram seems too big to be sent every `TaskInfo`
add `summarizeFinal` (see `io.trino.operator.TaskStats#summarizeFinal` for example) and only keep histogram there.

Reply:
done